addons/mail/static/src/messaging/entity/chat_window/chat_window.js

```diff
@@ -8,20 +8,6 @@ function ChatWindowFactory({ Entity }) {
 
     class ChatWindow extends Entity {
 
-        /**
-         * @override
-         */
-        delete() {
-            if (this.manager) {
-                this.manager.unregister(this);
-            }
-            const thread = this.thread;
-            super.delete();
-            if (thread) {
-                thread.update({ foldState: 'closed' });
-            }
-        }
```

Comment:
I don't think this is a good idea to remove this. The reason we need `register`/`unregister` is because we want to have full control of order of chat windows, and this is made possible with attribute field `_ordered`. By removing this, we can't manage order of chat windows, and that's a big issue because there are many features that rely on well-defined positioning (last visible, first visible, replace, swap, shift, just append, etc...)

Reply:
See reasoning for _order in other comment.

I will resolve this, but I will add reason to this one specifically:

It is important to not send the fold state to the server when just deleting entities (destroy, ...), only flux specific action should send it (user asked to close it).

Reply:
Agree delete of chat window doesn't mean change of fold state, it mixes causes and consequences and it's confusing.